tests/test_order_Hexatic.py

```diff
@@ -36,20 +36,20 @@ def test_compute_random(self):
         hop.compute((box, points))
         npt.assert_allclose(np.mean(hop.particle_order), 0.0 + 0.0j, atol=1e-1)
 
-    def test_compute(self):
+    @pytest.mark.parametrize("i", range(6))
+    def test_compute(self, i):
```

Comment:
This is not a parameterized test. This is a single test that constructs a single set of points using a `for` loop. All the points are intended to be in a single array. Please revert the changes in this test function.